app/controllers/product_drives_controller.rb

```diff
@@ -10,6 +10,8 @@ def index
                      .class_filter(filter_params)
                      .within_date_range(@selected_date_range)
                      .order(start_date: :desc)
+    # to be used in the name filter to sort product drives in alpha order
+    @product_drives_name_filter = @product_drives.sort_by { |pd| pd.name.downcase }
```

Comment:
Can we rename this to `@product_drives_alphabetical`? It's a bit more obvious what's inside it.

Reply:
updated.